decidim-core/db/migrate/20181008102144_add_badge_switch_to_organizations.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddBadgeSwitchToOrganizations < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_organizations, :badges_enabled, :boolean, null: false, default: false
```

Comment:
Why default false?

Reply:
I don't like `true` as a default for a boolean value in general, as it might be prone to errors as setting that value is not explicit. Which makes me think I forgot to set that value on organization creation!

Reply:
I'm not sure about this, to me it's kind of weird to have this as default false since by default we want it to be true.

Reply:
You can be explicit about it even if it's true at the database level.

Reply:
I'll break the tie: I'm with @josepjaume here